src/main/java/choose.java

```diff
@@ -0,0 +1,63 @@
+import java.util.Arrays;
+
+public class choose {
+    protected static Task[] list = new Task[100];
+    private static final int tasksAdded = 0;
```

Comment:
Like the use of variables to avoid magic literals! However, constant variables must be in all uppercase (using underscore to separate words).